tests/flow/test_enhanced_coverage.py

```
@@ -0,0 +1,293 @@
+        assert result == "OK"
+        
+        # Test listing constraints
+        result = self.db.execute_command("GRAPH.CONSTRAINT", "test_coverage", "LIST")
+        assert isinstance(result, list)
+        assert len(result) > 0
+        
+        # Test constraint violation
+        self.graph.query("CREATE (p:Person {id: 1, name: 'Alice'})")
+        try:
+            self.graph.query("CREATE (p:Person {id: 1, name: 'Bob'})")  # Should violate constraint
+            assert False, "Should have raised constraint violation"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)